st2actions/st2actions/runners/pythonrunner.py

```diff
@@ -165,6 +166,9 @@ def _get_output_values(self, exit_code, stdout, stderr, timed_out):
         else:
             error = None
 
+        if "InvalidStatusException" in stderr:
```

Comment:
That's not 100% fail-safe and robust so we need to be a bit careful (regular stdout could also contain this string).